### PR TITLE
week 2 notes

### PR DESCRIPTION
**Pull request by ajtowns**

### CODE REVIEW

**Comment by bitschmidty** [comment]
ACK 7a8884b

File: week-2.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+construction that allows to have a multitude of scripts as an alternative
+to the direct key path spend.
+
+## Results from Week One
+
+We had the first two Q&A sessions, and have logs of the conversations captured: [week 1 Q&A 1](http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-11-05-19.00.log.html),
+[week 1 Q&A 2](http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-11-07-02.00.log.html). There's some interesting discussion in both of them, so worth a look if you've got time!
+
+There's already been a few suggested changes to the BIPs, including [PR#122](https://github.com/sipa/bips/pull/122), [PR#126](https://github.com/sipa/bips/pull/126/files), and [PR#128](https://github.com/sipa/bips/pull/128/files) -- they're all pretty short, so if you haven't contributed to a BIP or Bitcoin core before, they make for pretty easy examples of how to contribute!
+
```
Comment (moneyball) [inline week-2.md:24]: I'd say this is the most significant thing arising from the week, so we should include it? https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-November/017443.html

Comment (ajtowns) [reply week-2.md:24]: makes sense!

**Comment by bitschmidty** [comment]
ACK e3741c7 , pending any last feedback on the poll for this week.

**Comment by harding** [comment]
Untested ACK, text LGTM.

**Comment by moneyball** [comment]
ACK

**Pull request merged**